ops/stg/persistent/okta.tf

```diff
@@ -1,9 +1,9 @@
 module "okta" {
   source               = "../../services/okta-app"
   env                  = local.env
-  redirect_urls        = []
   logout_redirect_uris = "https://${local.env}.simplereport.gov"
   app_url              = "https://${local.env}.simplereport.gov/app"
+  redirect_urls        = []
```

Comment:
I think this addition was supposed to be in prod, not stg?

Reply:
Apparently my find/replace wasn't good enough, should be resolved now.